File: Python/1 - Iniciante/1012.py
# coding: UTF - 8

A, B, C = input().split(" ")
A = float(A)
B = float(B)
C = float(C)

D = float((A * C) / 2)
E = float(3.14159 * (C * C))
F = float(((A + B) * C) / 2)
G = float(B * B)
H = float(A * B)

print("TRIANGULO: %.3f" % D)
print("CIRCULO: %.3f" % E)
print("TRAPEZIO: %.3f" % F)
print("QUADRADO: %.3f" % G)
print("RETANGULO: %.3f" % H)

File: Lua/Iniciante/1012.lua
-- Funcional no beecrowd
local A = io.read('n')
local B = io.read('n')
local C = io.read('n')

local triangulo = (A * C) / 2
local circulo = 3.14159 * (C * C)
local trapezio = ((A + B) * C) / 2
local quadrado = B ^ 2
local retangulo = A * B

print(string.format("TRIANGULO: %.3f", triangulo))
print(string.format("CIRCULO: %.3f", circulo))
print(string.format("TRAPEZIO: %.3f", trapezio))
print(string.format("QUADRADO: %.3f", quadrado))
print(string.format("RETANGULO: %.3f", retangulo))